en/docs/services/create-your-first-service.md

```diff
@@ -1,177 +1,260 @@
 
-- In the code view
+```ballerina
+import ballerinax/worldbank;
+import ballerinax/covid19;
+import ballerina/http;
 
-  ![Code View](../assets/img/services/choreo-service-code-view.png){.cInlineImage-full}
+service / on new http:Listener(8090) {
+    resource function get stats/[string country](http:Caller caller, http:Request request) returns error? {
+        covid19:Client covid19Client = check new ();
+        covid19:CovidCountry statusByCountry = check covid19Client->getStatusByCountry(country);
+        var totalCases = statusByCountry?.cases ?: 0d;
+        worldbank:Client worldBankClient = check new ();
+        worldbank:CountryPopulationArr? populationByCountry = check worldBankClient->getPopulationByCountry(country, 
+        "2019", format = "json");
+        var population = 
+        (populationByCountry is worldbank:CountryPopulationArr ? populationByCountry[0]?.value ?: 0 : 0) / 1000000;
 
-    
-To validate the service, click **Run & Test**. The following is logged to indicate that you have successfully started the service.
+        var totalCasesPerMillion = totalCases / population;
+        var payload = {
+            country: country,
+            totalCasesPerMillion: totalCasesPerMillion
+        };
 
-![Service Started Log](../assets/img/services/service-started-notification.png){.cInlineImage-half}
-    
-Now you are ready to test the service.
-   
-## Step 3: Test the service
+        check caller->respond(payload);
+    }
+}
+```
+
+## Step 6: Test the service
+
+To test the `CovidStatus` service you created, follow the procedure below:
 
-To test the `covid-stats` service, follow this procedure:
+1. Click **Run & Test**.
 
-1. Click the **Test** icon in the left pane.
+    The following logs appear to indicate that the service has successfully started.
+    
+    ```
+    Starting application...
+    [ballerina/http] started HTTP/WS listener 0.0.0.0:8090
+    ```
+   
+2. Click the **Test** icon in the left pane.
 
     ![Test Icon](../assets/img/services/test-icon.png){.cInlineImage-bordered}
 
-2. In the section that opens to the right of the page, click **GET**.
+3. In the test view that opens to the right of the page, click **GET**.
 
-3. Click **Try it out**, and then click **Execute**.
+4. Click **Try it out**, and in the **country** field, enter `USA`.
+   
+5. Click **Execute**.
 
-The following figure shows the server response to the search results for the United States COVID-19 statistics:
+    The response is displayed as follows:
 
-![Server Response](../assets/img/services/server-response.png){.cInlineImage-half}
+    ![Response in the test view](../assets/img/services/test-view-response.png){.cInlineImage-bordered}
 
-Now that you have verified that the `covid-stats` service works as expected, you can deploy it.
 
-## Step 4: Deploy the service
+## Step 7: Deploy the service
 
-To deploy the `covid-stats` service, follow this procedure:
+To deploy the `CovidStatus` service, follow these steps:
```

Comment:
In some places we say "follow these steps:" and there are places where we have used "follow the procedure below:". Would be good to follow a consistent style throughout the doc.

Reply:
Changed "Steps" to "Procedure" except in the introductory text. Did not change that because we refer to the hight level steps there.